felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -223,6 +226,9 @@ func newBPFEndpointManager(
 		hostIfaceToEpMap: map[string]proto.HostEndpoint{},
 		ifaceToIpMap:     map[string]net.IP{},
 		opReporter:       opReporter,
+		// ipv6Enabled Should be set to config.IPv6Enabled, but for now it is better
+		// to set it to false since IPv6 supoprt is not yet implemented
+		ipv6Enabled: false,
```

Comment:
I think we should include an FV test that would set this to `true` so that we know that the skeletons we have load. Those skeletons don't do much yet, but imo we want to avoid merging code that is never executed. Perhaps we should also have a simple UT test that would parse a IPv4 and IPv6 packets when the feature is enabled. Even if the skeleton just accepts the packet or something like that without doing much more. WDYT?